Documentation/Presentations.md

```diff
@@ -0,0 +1,224 @@
+# How to design and prototype custom presentation animation in Interface Builder with IBAnimatable
+
+For most of the non-trivial apps, we have to present modally some Scene (`ViewController`). August 20th, 2016 as of this writing, out of the box, Interface Builder only supports fullscreen modals with limited animations e.g. "Cover Vertical" and "Flip Horizontal". Using `IBAnimatable` we can configure in the interface builder the presentation animation, the dismissal animation, the modal size, the `dimmingView` appearance, and a lot more.
```

Comment:
For most of the non-trivial apps, we have to present some modal view controller (Scene in Interface Builder). August 20th, 2016 as of this writing, out of the box, Interface Builder only supports fullscreen modals with limited animations e.g. "Cover Vertical" and "Flip Horizontal". Using `IBAnimatable` we can configure how to present a modal in the interface builder, for example, the presentation animation, the dismissal animation, the modal size, the `dimmingView` appearance, and a lot more.